src/components/general/SideSheet/Standard/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, useState, useCallback, useRef, useEffect, ReactNode } from 'react';
+import classNames from 'classnames';
+import { useComponentDisplayClassNames } from '@equinor/fusion';
+import { useEventListener } from '@equinor/fusion-components';
+import styles from './styles.less';
+import CollapseExpandButton from './CollapseExpandButton';
+import { OverlayPortal, Scrim } from '@equinor/fusion-components';
+
+export type StandardSideSheetProps = {
+    id: string;
+    title?: string;
+    isOpen: boolean;
+    onOpenChange: (isOpen: boolean) => void;
```

Comment:
Should this be onClose instead? The presenter controls when it opens, but needs to be notified when it's closed